source/patterns/@aws-solutions-constructs/aws-openapigateway-lambda/lib/index.ts

```diff
@@ -11,58 +11,17 @@
  *  and limitations under the License.
  */
 
-import { Aws } from 'aws-cdk-lib';
 import * as cdk from "aws-cdk-lib";
 import { Construct } from 'constructs';
 import * as apigateway from 'aws-cdk-lib/aws-apigateway';
-import * as lambda from 'aws-cdk-lib/aws-lambda';
 import * as logs from 'aws-cdk-lib/aws-logs';
 import * as iam from 'aws-cdk-lib/aws-iam';
 import * as s3 from 'aws-cdk-lib/aws-s3';
 import * as defaults from '@aws-solutions-constructs/core';
-import * as resources from '@aws-solutions-constructs/resources';
 import { RestApiBaseProps } from 'aws-cdk-lib/aws-apigateway';
 import { Asset } from 'aws-cdk-lib/aws-s3-assets';
-
-/**
- * The ApiIntegration interface is used to correlate a user-specified id with either a existing lambda function or set of lambda props.
- *
- * See the 'Overview of how the OpenAPI file transformation works' section of the README.md for more details on its usage.
- */
-export interface ApiIntegration {
-  /**
-   * Id of the ApiIntegration, used to correlate this lambda function to the api integration in the open api definition.
-   *
-   * Note this is not a CDK Construct ID, and is instead a client defined string used to map the resolved lambda resource with the OpenAPI definition.
-   */
-  readonly id: string;
-  /**
-   * The Lambda function to associate with the API method in the OpenAPI file matched by id.
-   *
-   * One and only one of existingLambdaObj or lambdaFunctionProps must be specified, any other combination will cause an error.
-   */
-  readonly existingLambdaObj?: lambda.Function;
-  /**
-   * Properties for the Lambda function to create and associate with the API method in the OpenAPI file matched by id.
-   *
-   * One and only one of existingLambdaObj or lambdaFunctionProps must be specified, any other combination will cause an error.
-   */
-  readonly lambdaFunctionProps?: lambda.FunctionProps;
-}
-
-/**
- * Helper object to map an ApiIntegration id to its resolved lambda.Function. This type is exposed as a property on the instantiated construct.
- */
-export interface ApiLambdaFunction {
-  /**
-   * Id of the ApiIntegration, used to correlate this lambda function to the api integration in the open api definition.
-   */
-  readonly id: string;
-  /**
-   * The instantiated lambda.Function.
-   */
-  readonly lambdaFunction: lambda.Function;
-}
+import { ApiIntegration, CheckOpenapiProps, TokenToFunctionMapping, ObtainApiDefinition } from './openapi-helper';
+export { ApiIntegration, TokenToFunctionMapping as ApiLambdaFunction } from './openapi-helper';
```

Comment:
As we don't normally export imports? in this library, could we add a comment here explaining the why and how its used?